lib/model/observable/observable.dart

```diff
@@ -278,6 +307,14 @@ class ObservableReference<T> extends ChangeNotificationProvider<T>
     return _listenSub.asFuture();
   }
 
+  /// A [Stream] of all non-null values on this view, including the current
+  /// [value] at the time the stream is listened to (if it's non-null).
+  ///
+  /// This implementation will never emit equal consecutive data events.
+  /// "Equality" is defined by [equalsFn].
```

Comment:
In the code the private `_equalsFn` is used. Is there a public `equalsFn` as well? If not, then this shouldn't be a doc link.

Reply:
Doesn't look like there is a public getter or anything by that name, but it is an optional argument on the constructor. Does dartdoc link to that or is it creating a dead link?

Reply:
When it can't find a doc link, dartdoc renders the entry as code. So the docs won't have a dead link, but it may still puzzle the reader. Maybe replace `[equalsFn]` by the description you gave. Something like:

> "Equality" is defined by the [EqualsFn] passed to the constructor.

`EqualsFn` is a public typedef, so that reference is ok.